### PR TITLE
feat: badge

### PR DESCRIPTION
**Pull request by shashidwivedi**
Badge component on icons and avatars:
image

### CODE REVIEW

File: styles/foobar-ui.css [hunk 1]
```diff
@@ -508,7 +508,37 @@ template {
...
+  border-radius: calc(var( --space-xs) / 2);
+  line-height: calc(var( --space-xs) / 2);
+  height: var( --space-xs);
+  width: var( --space-xs);
+  background-color: var(--color-secondary);
+  color: var(--color-white);
+  padding: 0 4px;
+}
 
 .btn {}
```
Comment (Sameony) [inline styles/foobar-ui.css:543]: do remove empty blocks

File: styles/foobar-ui.min.css [hunk 2]
```diff
@@ -1 +1 @@
...
...ar(--border-radius-circle);background-color:var(--color-secondary);color:var(--color-black);font-weight:500;display:flex;justify-content:center;align-items:center}.avatar-sm{height:var(--space-lg);width:var(--space-lg);font-size:var(--space-xs)}.avatar-md{height:var(--space-xl);width:var(--space-xl);font-size:var(--space-sm)}.avatar-lg{height:var(--space-xxl);width:var(--space-xxl);font-size:var(--space-md)}.avatar__img{width:100%;height:100%;object-fit:cover;border-radius:var(--border-radius-circle)}.badge{position:relative;font-size:var(--font-size-md)}.badge-anchor{position:absolute;top:calc(var(--font-size-xs) * -1);right:calc(var(--font-size-sm) * -1);font-size:calc(var( --font-size-md) / 2);border-radius:calc(var(--space-xl)/ 2);line-height:calc(var( --space-xl) / 2);height:calc(var(--space-xl)/ 2);min-width:calc(var(--space-xl)/ 2);background-color:var(--color-primary);color:var(--color-white);padding:0 6px}.badge-anchor-avatar{position:absolute;bottom:0;right:var(--space-xs);border-radius:calc(var(--space-xs)/ 2);line-height:calc(var( --space-xs) / 2);height:var(--space-xs);width:var(--space-xs);background-color:var(--color-secondary);color:var(--color-white);padding:0 4px}
```
Comment (Sameony) [inline styles/foobar-ui.min.css:1]: give proper indentation and alignment to the code blocks.
this is unreadable

**Review by Sameony — APPROVED** [review]
Do focus more on alignment and indentation

**Review by tanishq20 — APPROVED** [review]
Very well-written code. Impressed with your CSS code.

File: styles/foobar-ui.min.css (hunk 2 above)
Comment (shashidwivedi) [reply styles/foobar-ui.min.css:1]: Hi @Sameony , this is actually minified file.

File: styles/foobar-ui.css (hunk 1 above)
Comment (shashidwivedi) [reply styles/foobar-ui.css:543]: Yes, thanks for the suggestion. Keeping it as placeholder for upcoming styles

**Pull request merged**